templates/frontend/shared/start.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+CONTRACT_DIRECTORY=../contract
+DEV_ACCOUNT_FILE="${CONTRACT_DIRECTORY}/neardev/dev-account.env"
+
+start () {
+  echo The app is starting!
+  env-cmd -f $DEV_ACCOUNT_FILE parcel index.html --open
+}
+
+alert () {
+  echo "======================================================"
+  echo "Can't find the dev-account credentials file (${DEV_ACCOUNT_FILE})"
+  echo "Did you deploy your contract ?"
+  echo ">> Run 'npm run deploy' from your 'contract' directory"
+  echo "======================================================"
```

Comment:
@gagdiez wdyt about the message

Reply:
I would recommend the 'root' directory, since people will be most probably there, and the `package.json` is ready for that.

Reply:
suggestion: 

```bash
GREEN='\033[1;32m'
NC='\033[0m' # No Color

echo "======================================================"
echo "It looks like you forgot to deploy your contract"
echo ">> Run ${GREEN}'npm run deploy'${NC} from the 'root' directory"
echo "======================================================"
```